Source Code/Ticket Management System/ticket.c

```diff
@@ -268,6 +270,11 @@ wchar_t* ticket_getTitle(struct Ticket* ticket)
 	return NULL;
 }
 
+wchar_t* ticket_getMedia(struct Ticket* ticket)
```

Comment:
Don't commit temporary things. This will bite you in the form of a merge conflict.

Reply:
How do I cancel the changes to ticket.h/.c?

Reply:
You do the opposite changes.